src/css/style.css

```diff
@@ -72,4 +73,115 @@
 .map-icon {
 	position: relative;
 	top: 2px;
+}
+
+footer {
+	height: auto;
+	background-color: #006400;
+	width: 100%;
+	padding-top: 20px;
+	align-items: center;
+	justify-content: space-around;
+}
+
+small {
+	color: #fff;
+	text-align: center;
+}
+
+.footer-support-text-owned {
+	font-size: 10px;
+	padding-top: 20px;
+	margin-top: 20px;
+}
+
+.footer-container {
+	padding-right: 15px;
+	padding-left: 15px;
+	margin-right: auto;
+	margin-left: auto;
+	padding-top: 15px;
+	margin-top: 15px;
+	grid-template-columns: repeat(auto-fit, 100px);
+}
+
+.column {
+	float: left;
```

Comment:
I see you're using the CSS grid, maybe use CSS grid instead of floats. [This](https://learncssgrid.com/) might help you if you're not familiar with CSS Grid.